pyigm/cgm/utils.py

```diff
@@ -256,19 +257,22 @@ def cgm_from_galaxy_igmsystems(galaxy, igmsystems, rho_max=300*u.kpc, dv_max=400
             cgm = CGMAbsSys(galaxy, dummysystem, cosmo=cosmo, **kwargs)
             cgm_list = [cgm]
     else:
-        from linetools.analysis.zlimits import zLimits
         # Loop to generate
         cgm_list = []
         for imatch in match:
             # Instantiate new IGMSystem
             # Otherwise, updates to the IGMSystem cross-pollinate other CGMs
             sysmatch = igmsystems[imatch]
             newisys = sysmatch.copy()
+            # Handle z limits
             zlim = ltu.z_from_dv((-dv_max.value,dv_max.value)*u.km/u.s,galaxy.z)
             newlims = zLimits(galaxy.z,zlim.tolist())
             newisys.limits = newlims
+            # Allow for components extending beyond dv_max
+            newisys.update_vlim()
```

Comment:
Ha! I see I misunderstood the docstring for this method in my linetools PR review.